influx-writer/influx-writer.py

```diff
@@ -139,9 +141,15 @@ def start(self):
                 if self.time > 0:
                     self.log.error("Inserted %d rows in %.1fs (%.2f listens/sec). Total %d rows." % \
                         (count, self.time, count / self.time, self.total_inserts))
+                    print("Inserted %d rows in %.1fs (%.2f listens/sec). Total %d rows." % \
```

Comment:
This wasn't an error so I didn't use `print_and_log_error` here, but changed now.